### ISSUE TITLE
Error generating default config files

### ISSUE THREAD

**Issue by bhainesva** [TypicalAM/goread#11]
Hello, I tried to install and run `goread` with the following commands:

```
go install github.com/TypicalAM/goread@latest
goread
```

I got this error from the second command:
```
Failed to load the cache: open /Users/bhaines/Library/Caches/goread/cache.json: no such file or directory, creating a new oneThere has been an error executing the commands: 'open /Users/bhaines/Library/Application Support/goread/urls.yml: no such file or directory'%
```

It looks like the `/Users/bhaines/Library/Caches/goread` and `/Users/bhaines/Library/Application Support/goread` directories weren't created. After creating them manually `goread` had this new error:

```
Failed to load the cache: unexpected end of JSON input, creating a new oneThere has been an error executing the commands: 'open /Users/bhaines/Library/Application Support/goread/urls.yml: no such file or directory'%                                                                                                                                                                                 
```

After manually creating the cache.json and urls.yml files `goread` runs successfully.

I'm using `go version go1.20.6 darwin/amd64` on macOS Monterey 12.3. 

Thanks!

**Comment by TypicalAM** [comment]
Hi, should be working as of now (pr #13 ). Will update release version soon

**Comment by TypicalAM** [comment]
Fixed in [v1.3.2](https://github.com/TypicalAM/goread/releases/tag/v1.3.2)
